### PR TITLE
[Multi-tenancy/namespace consumer group] Create howto and add to overview

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

- Add namespace consumer groups to overview
- Create howto doc for using namespace value

## Issue reference

PR will close: #2769

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  metadata:
+  - name: redisHost
+    value: localhost:6379
+  - name: redisPassword
+    value: ""
+  - name: consumerID
+    value: "{namespace}"
+```
+
+By calling `consumerID` with the `{namespace}` value, you'll be able to use the same `app-id` with the same topics from different namespaces.
```
Comment (yaron2) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md]: Maybe change `calling` to `configuring`?

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+<img src="/images/howto-namespace/with-namespace.png" width=1000 alt="Diagram showing how namespace consumer groups help with multi-tenancy.">
+
+In the diagram above, you have two namespaces, each with applications of the same `app-id`, publishing and subscribing to the same centralized message bus `orders. This time, however, Dapr has created consumer group names prefixed with the namespace in which they're running.
```
Comment (yaron2) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md]: typo: insert missing ` after orders.

**Review by yaron2 — CHANGES_REQUESTED** [review]

**Review by yaron2 — APPROVED** [review]
LGTM

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+In the diagram above, you have two namespaces, each with applications of the same `app-id`, publishing and subscribing to the same centralized message bus `orders`. This time, however, Dapr has created consumer group names prefixed with the namespace in which they're running. 
+
+Without you needing to change your code/`app-id`, the namespace consumer group allows you to:
+- Add more namespaces
+- Keep the same topics
+- Keep the same `app-id` across namespaces
+- Have your entire deployment pipeline remain intact
+
+Simply include the `"{namespace}"` consumer group construct in your component metadata. You don't need to encode the namespace in the metadata. Dapr will understand the namespace running and will fill the namespace for you, like a dynamic metadata value injected by the runtime.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md]:
```suggestion
Simply include the `"{namespace}"` consumer group construct in your component metadata. You don't need to encode the namespace in the metadata. Dapr understands the namespace it is running in and completes the namespace value for you, like a dynamic metadata value injected by the runtime.
```

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+{{% alert title="Note" color="primary" %}}
+If you add the namespace consumer group to your metadata afterwards, Dapr will update everything for you.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md]:
```suggestion
If you add the namespace consumer group to your metadata afterwards, Dapr updates everything for you. This means that you can add namespace metadata value to existing pub/sub deployments.
```

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+## Demo
+
+Watch [this video for an overview on pub/sub multi-tenancy](https://youtu.be/eK463jugo0c?t=1188):
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/eK463jugo0c?start=1188" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Next steps
+
+- Learn more about [pub/sub namespaces]({{< ref pubsub-namespaces >}}).
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-namespace.md]:
```suggestion
- Learn more about configuring Pub/Sub components with multiple namespaces [pub/sub namespaces]({{< ref pubsub-namespaces >}}).
```

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-overview.md [hunk 6]
```diff
@@ -96,6 +96,10 @@ For more information on message routing, read [Dapr pub/sub API reference]({{< r
 
 Sometimes, messages can't be processed because of a variety of possible issues, such as erroneous conditions within the producer or consumer application or an unexpected state change that causes an issue with your application code. Dapr allows developers to set dead letter topics to deal with messages that cannot be delivered to an application. This feature is available on all pub/sub components and prevents consumer applications from endlessly retrying a failed message. For more information, read about [dead letter topics]({{< ref "pubsub-deadletter.md">}})
 
+### Namespace consumer groups
+
+Dapr solves multi-tenancy at-scale with its [namespace consumer groups construct]({{< ref howto-namespace >}}). Simply include the `"{namespace}"` value in your component metadata to allow multiple namespaces with applications of the same `app-id` to publish and subscribe to the same centralized message bus.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-overview.md]:
```suggestion
Dapr solves multi-tenancy at-scale with [namespaces for consumer groups]({{< ref howto-namespace >}}). Simply include the `"{namespace}"` value in your component metadata for consumer groups to allow multiple namespaces with applications of the same `app-id` to publish and subscribe to the same message broker.
```

**Comment by msfussell** [comment]
@hhunter-ms - The with-namespace.png file still stays "message bus"

**Pull request merged**